chia/server/start_wallet.py

```diff
@@ -8,7 +8,7 @@
 from chia.server.outbound_message import NodeType
 from chia.server.start_service import run_service
 from chia.types.peer_info import PeerInfo
-from chia.util.block_tools import test_constants
```

Comment:
this dependency on the test constants (in the production wallet) seems a bit odd